net-p2p/qbittorrent/qbittorrent-4.0.4.ebuild

```diff
@@ -43,7 +43,6 @@ DEPEND="${RDEPEND}
 
 DOCS=( AUTHORS Changelog CONTRIBUTING.md README.md TODO )
 PATCHES=( "${FILESDIR}/${PN}-4.0.4-werror.patch" )
-PATCHES=( "${FILESDIR}/${PN}-4.0.4-musl.patch" )
```

Comment:
I don't understand what's going on here.  Why are you removing the musl patch from 4.0.4?